test/kubernetes_unittest.cc

```diff
@@ -0,0 +1,72 @@
+#include "../src/configuration.h"
+#include "../src/instance.h"
+#include "../src/kubernetes.h"
+#include "../src/updater.h"
+#include "gtest/gtest.h"
+
+namespace google {
+
+class KubernetesTest : public ::testing::Test {
+  protected:
```

Comment:
Please indent access modifiers by 1 space, and the class body by 2 spaces ([style guide](https://google.github.io/styleguide/cppguide.html#Class_Format)).

Reply:
Fixed